ko/lessons/specifics/ecto.md

```diff
@@ -9,6 +11,13 @@ Ecto는 공식적인 Elixir 프로젝트로 데이터베이스를 감싸는 부
 
 ## 설정하기
 
+슈퍼바이저 트리를 포함한 프로젝트 생성:
```

Comment:
Can you make these sentenses to be more finished-looking? like
> 슈퍼바이저 트리를 포함해서 새 애플리케이션을 생성합니다.

Reply:
Thank you for the guide! I updated all following your guide including the below job.